gnome-initial-setup/gis-page.h

```diff
@@ -39,6 +39,8 @@ typedef struct _GisPage        GisPage;
 typedef struct _GisPageClass   GisPageClass;
 typedef struct _GisAssistantPagePrivate GisAssistantPagePrivate;
 
+G_DEFINE_AUTOPTR_CLEANUP_FUNC (GisPage, g_object_unref)
```

Comment:
Why is this needed? Where is the autoptrness used? If this is a speculative change for future refactoring & upstreaming, let's leave it out for now.